CMakeLists.txt

```diff
@@ -237,8 +214,7 @@ list(TRANSFORM AVR_SOURCES PREPEND ${PRUSA_BOARDS_DIR}/cores/prusa_einsy_rambo/)
 # Target configuration
 #
 if(CMAKE_CROSSCOMPILING)
-  # set source epoch
-  set_source_epoch(${PROJECT_VERSION_TIMESTAMP})
```

Comment:
This has been changed by https://github.com/prusa3d/Prusa-Firmware/commit/cfc8ffe8a36ba3583b2b7038c48a02afd079106b What is correct?